FF8.Core/Sources/FieldName.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace FF8.Core
+{
+    public static class FieldName
+    {
+        public static String Get(FieldId id)
+        {
+            if (_dic.TryGetValue(id, out var name))
+                return name;
+
+            return $"Unknown field: {id}";
+        }
+
+        private static readonly Dictionary<FieldId, String> _dic = new Dictionary<FieldId, String>()
+        {
+            {FieldId.bghoke_2, "B-Garden - Infirmary"}
```

Comment:
TODO: Add the names
@Sebanisu - do you remember where the location names are? Did we actually find that thing?

Reply:
yes they are pulling out of Areames and namedic

Reply:
There is a LocationID, Currentfield and Previousfield in the save. They are different values. So I'm unsure if there is a conversion for feild to locationID.

Reply:
There are only 250 location strings and 981 fieldIDs

Reply:
Thanks- didn't know that. I wonder how they are all connected to each other (grouped, like how the engine knows which location string is what field)

Reply:
He has the Location names hard coded under AreaName.cs.